src/compiler/sys.ts

```diff
@@ -184,7 +192,7 @@ namespace ts {
 
                 function fileEventHandler(eventName: string, relativeFileName: string, baseDirPath: string) {
                     // When files are deleted from disk, the triggered "rename" event would have a relativefileName of "undefined"
-                    const fileName = typeof relativeFileName !== "string"
+                    const fileName = !isString(relativeFileName)

```

Comment:
The input type is `string`, so this should always be a string?

Reply:
it can be undefined, updated definition accordingly

Reply:
Would be simpler to just check `=== undefined` instead of `!isString`, if it can only be `string | undefined`.